docs/tutorial/twitter/tutorial-part-2.md

```diff
@@ -258,7 +258,7 @@ we need to run the compiler once more. After that, the image should be in the
 windows caption bar.
 
 Two more minor things are left to finish. First, the button does not look very
-good. Why don't we just give it a fixed width to fit its height.
+good. Why don't we just give it a fixed width to fit its height?
```

Comment:
This is a rhetorical question, not really a question. I'd either leave it as a period, or replace the period with a colon (`:`)